### DISCUSSION TITLE
Logic Pro X to OBS via (BlackHole + Clarrett)

### DISCUSSION THREAD

**Discussion by MichellePackman** [ExistentialAudio/BlackHole#96] [General]
Hey Devin, thanks for all of your work so far with BlackHole!

I'm on High Sierra 10.13.6 and use Logic Pro X 10.4.8. I'm a music producer, I use BlackHole to stream my musical performances on Twitch. 

While streaming, I have been having issues with deteriorating signal clarity from Logic to OBS Studio via BlackHole + Clarrett interface as an aggregate device. I'm using my Clarrett for monitoring and my input in OBS is just BlackHole. 

The signal quality in OBS usually starts out fine but eventually it deteriorates and starts to have a lot of digital noise, sounds similar to analog white noise on old TV sets. I have to reset this often (every 3-5 mins or so) in OBS and sometimes it takes 30-40 seconds from the time of assigning the input in OBS for the signal to start appearing in the meter and for people watching to be able to hear sound. 

Additionally, if there is any kind of OBS application that works best with BlackHole, please advise!

Thanks! Michelle

**Comment by devinroth** [comment]
That sounds like it may be clock drifting. Make sure you have drift compensation enabled. I’ve been streaming from Logic to Zoom for six hours a day without any issues. 

How exactly do you have everything setup? Screenshots are helpful. 

Devin
